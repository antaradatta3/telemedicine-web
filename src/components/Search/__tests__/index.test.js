import React from 'react';
import { mount } from 'enzyme';

import Search from '../index';

describe('Component: Search', () => {
  const props = {
    label: 'Sample',
    type: 'primary',
    count: 200,
  };
  it('Search matches snapshot', () => {
    const component = mount(<Search {...props} />);
    expect(component).toMatchSnapshot();
  });
});
