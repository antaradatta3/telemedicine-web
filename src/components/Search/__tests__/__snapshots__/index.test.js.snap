// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Search Search matches snapshot 1`] = `
<Search
  count={200}
  label="Sample"
  type="primary"
>
  <div>
    <SearchForm
      search={[Function]}
    >
      <div
        className="form-inline"
      >
        <input
          className="form-control"
          onChange={[Function]}
          placeholder="Search Thread"
          type="text"
          value=""
        />
        <button
          className="btn btn-primary"
          onClick={[Function]}
          type="button"
        >
          Search
        </button>
      </div>
    </SearchForm>
  </div>
</Search>
`;
